cmake/gen/rvv_microkernels.cmake

```diff
@@ -44,8 +44,12 @@ SET(PROD_RVV_MICROKERNEL_SRCS
   src/f32-vlrelu/gen/f32-vlrelu-rvv-u4v.c
   src/f32-vrelu/gen/f32-vrelu-rvv-u4v.c
   src/f32-vrsqrt/gen/f32-vrsqrt-rvv-rsqrt-u4v.c
+  src/qs8-gavgpool/gen/qs8-gavgpool-7p7x-minmax-fp32-rvv-c2v.c
```

Comment:
c2v function/file naming inconsistent with rvv kernels
should be u2v meaning unrolled to 2 vectors.. aka m2

Reply:
f32 gavgpool uses the old style naming as well, should get fixed